README.md

```diff
@@ -1,2 +1,76 @@
-# ts_m2fpga
-M2 Controller FPGA code
+# M2 Cell Controller FPGA Code
+
+## Overview
+
+This project will improve the FPGA code in the [M2 Cell Controller LaVIEW project](https://github.com/lsst-ts/ts_mtm2_cell).
+In that version, the VI named **4-port serial Master Slave.vi**, under `Chassis (cRIO-9038) >> FPGA Target (RIO0, cRIO-9038)` in the Project Explorer, interacts with the hardware inputs and outputs.
+
+## Directory Structure
+
+```bash
+ts_m2fpga
+├── src
+│   ├── m2fpga.lvproj
+│   ├── mainFPGA.vi
+│   ├── moduleRelays.vi
+│   ├── moduleDaq.vi
+│   ├── moduleIlc.vi
+├── doc
+│   ├── versionHistory.md
+│   ├── moduleRelays.md
+│   ├── moduleDaq.md
+│   ├── moduleIlc.md
+├── dependencies
```

Comment:
The `dependencies` is a reserved word to say this module depends on "what" libraries, modules, etc. Here, they are the output from the LabVIEW code generator. They are the dependencies for `ts_m2cellcpp` but not this module. Consider rename to `fpgaCode` or something similar.

Reply:
Changed `dependencies` to `fpgaInterface` directory name.